pkg/ipmi/ipmi_dev.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package ipmi
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"runtime"
+	"syscall"
+	"time"
+	"unsafe"
+
+	"golang.org/x/sys/unix"
+)
+
+type Dev struct {
+	*os.File
+}
+
+var (
```

Comment:
Instead of these functions being globals, could they be private fields on the Dev struct?

Reply:
Your idea as well as my implementation have their own drawbacks. Yours is more logical on a structural level, but prolly not so nice to read and mine is the opposite. not so logical on the structural level, but better to read.

I would assume both is not good go style, but maybe I miss something. I"m undecided. Maybe you have a good argument for your idea?

Reply:
Ok, never mind. I see the prob here with glob vars